compiler/kernel/vop.go

```diff
@@ -150,12 +152,14 @@ func (b *Builder) compileVamLeaf(o dag.Op, parent vector.Puller) (vector.Puller,
 		}
 		return vamop.NewSort(b.rctx, parent, sortExprs, o.NullsFirst, o.Reverse, b.resetters), nil
 	case *dag.Summarize:
-		if name, ok := optimizer.IsCountByString(o); ok {
-			return vamop.NewCountByString(b.zctx(), parent, name), nil
-		} else if name, ok := optimizer.IsSum(o); ok {
-			return vamop.NewSum(b.zctx(), parent, name), nil
+		// XXX This only works if the key is a string which is unknowable at
```

Comment:
Please uncomment this so I can run in demo.  It's only a problem if you put vectors in a pool which none of our users are doing yet.

Reply:
The problem with this is it causes count by any single argument type other than string to fail. I disabled this because it was causing these tests to fail. I already have a separate branch that adds this back but in runtime. How about we merge this and post up this branch for review shortly after?